README.md

```diff
@@ -1675,6 +1676,33 @@ aem {
 
 There are also available convenient methods `asStream`, `asString` to be able to process endpoint responses.
 
+#### Making changes to repository
+
+To make some changes in repository on AEM instance use `nodes` namespace in `aem.sync`
+
+```kotlin
+aem {
+    tasks {
+        instanceSatisfy {
```

Comment:
this example is bad, because you are assuming `instanceSatisfy` and `packageDeploy` within single gradle execution. they could be launched separately and then DISABLED state will remain on instance unintentionally

Reply:
I was trying to make this in single task by using `doFirst` and `finalize` but it didin't work as expected.

Reply:
tomorrow will be fixed / docs